src/containers/Tasks/Tasks.js

```diff
@@ -35,46 +35,44 @@ import {
 } from '@carbon/icons-react';
 
 import { ListPageLayout } from '..';
-import { fetchTasks } from '../../actions/tasks';
+import { fetchTasks as fetchTasksActionCreator } from '../../actions/tasks';
 import { deleteTask } from '../../api';
 import {
   getSelectedNamespace,
   getTasks,
   getTasksErrorMessage,
   isFetchingTasks,
-  isReadOnly,
-  isWebSocketConnected
+  isWebSocketConnected,
+  isReadOnly as selectIsReadOnly
 } from '../../reducers';
 
-export /* istanbul ignore next */ class Tasks extends Component {
-  state = {
-    deleteError: null,
-    showDeleteModal: false,
-    toBeDeleted: []
-  };
+/* istanbul ignore next */
+function Tasks(props) {
+  const {
+    filters,
+    namespace,
+    fetchTasks,
+    error,
+    intl,
+    tasks,
+    loading,
+    isReadOnly,
+    namespace: selectedNamespace,
+    webSocketConnected
```

Comment:
nit: these should be in alphabetical order so they're easier to read / quickly scan